### ISSUE TITLE
[Question]: Why images not get sent to the model even though the image-to-text model is set up

### ISSUE THREAD

**Issue by cyhasuka** [infiniflow/ragflow#2585]
### Describe your problem

Why images not get sent to the model even though the image-to-text model is set up? 
For now, pics will sent to Knowledge base and run OCR, and calling CV LLM when OCR does not recognise enough text. This makes sense if the model set in the Chat page is LLM. But when the model set in the page is CV LLM, Pics should be sent directly to the model.
![image](https://github.com/user-attachments/assets/dabc3968-cd79-4997-b4d4-7d6f905e4262)

**Comment by KevinHuSh** [comment]
You need set it up here:
![image](https://github.com/user-attachments/assets/07e6f0c6-1af5-4939-b441-e9d8e533f176)

**Comment by cyhasuka** [comment]
Yes, I already setup here, but problem continued. Pics  has not been
directly sent to cv llm.

Kevin Hu ***@***.***>于2024年9月25日 周三19:22写道：

**Comment by KevinHuSh** [comment]
By our evaluation, without OCR,  multi-modal LLM can't capture detail information for most cases, especially when there's text on it.

**Comment by cyhasuka** [comment]
For example, this hf spaces [Qwen2-VL](https://huggingface.co/spaces/Qwen/Qwen2-VL) asks for an image to be entered along with the prompt, and the result is output after processing.

According to our tests, the image to text models (e.g., qwen2-VL, GPT-4o) nowadays have much better image to text understanding performance than the dedicated OCR models in ragflow. Pictures that contain both text and geometric image features cannot be accurately understood by OCR. 

Therefore, we believe that if a user specifies a CVLLM model in a dialogue, the picture should be sent to the model along with the prompt words.

**Comment by Jas0nxlee** [comment]
same to u.

**Comment by Jas0nxlee** [comment]
I also have the same problem and hope to send images from the knowledge base to a multimodal model.

**Comment by KevinHuSh** [comment]
> I also have the same problem and hope to send images from the knowledge base to a multimodal model.

If OCR can't extract text from picture, it will be sent to multimodal model.
Is that okay for you?

**Comment by cyhasuka** [comment]
> > I also have the same problem and hope to send images from the knowledge base to a multimodal model.
> 
> If OCR can't extract text from picture, it will be sent to multimodal model. Is that okay for you?

No. If a user specifies a CVLLM model in a dialogue, the picture should be sent to the model along with the prompt words. 

To use an example of an application scenario, if I send a photo with an advertising board in the background of the photo. Then OCR will only recognize the text information in the advertisement plate, which I don't need. I need LLM to understand this image, not simply extract the text. Otherwise, it will be pointless for me to deploy CVLLM in chat.

**Comment by cyhasuka** [comment]
At the very least, the user can be given a choice on the front-end page.

**Comment by bored-again** [comment]
Having the same problem. Is there a fix?

**Comment by which-W** [comment]
@cyhasuka 
Thanks so much for the suggestion — and apologies for the delayed reply! ⏳

At the moment, we don’t support an option to use only LLMs for CV-related tasks. That said, we appreciate the idea and will consider its potential value for future development 🔍💡.

Feel free to keep an eye on the project by giving it a ⭐, and we’d love to hear more of your constructive thoughts anytime!

Thanks again! 🙌

**Comment by which-W** [comment]
@cyhasuka 
Thanks a ton for the thoughtful suggestion — and sorry for the late reply! 🕐

Currently, we don’t offer an option to process images using LLM-only CV functionality. However, your idea makes a lot of sense, and we’ll take it into consideration as we continue to evaluate what features could add the most value to our users 💡📷.

Feel free to ⭐ the project to stay up to date. And as always, we really appreciate any other constructive feedback you might have!

Thanks again for being part of the community! 🙌

**Comment by decaMinCow** [comment]
可以了吗
